### PR TITLE
Partner Manager 2.8

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -7,13 +7,20 @@
...
+* xref:configure-partner.adoc[Configure and Manage Trading Partners]
+  ** xref:create-partner.adoc[Create a New Partner]
+  ** xref:modify-partner-settings.adoc[Modify a Partner's Settings]
   ** xref:partner-manager-delete-partner.adoc[Delete a Partner]
-* xref:partner-manager-configuration-objects.adoc[Configuration Objects]
+  * xref:configure-third-party.adoc[Configure and Manage 3rd Party Connections]
+    ** xref:create-third-party.adoc[Create a New 3rd Party Connection]
+    ** xref:modify-third-party-settings.adoc[Modify a 3rd Party Connection's Settings]
+    ** xref:delete-third-party.adoc[Delete a 3rd Party Connection]
+ * xref:configure-send-receive-settings.adoc[Configure Send and Receive Settings for Partners and 3rd Party Connections]
```
Comment (vijayang) [inline modules/ROOT/nav.adoc]: 3rd party connections do not have X12/EDIFACT settings. 
The label for this should just be "Configure Send and Receive Settings for Partners"

File: modules/ROOT/pages/create-third-party.adoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
++
+[%header%autowidth.spread]
+|===
+| Setting a| Description
+| *Name*
+| Text string that easily identifies the connection
+| *Type*
+| Identifier type
+| *Identifier Value*
+| Value that uniquely identifies the connection
```
Comment (vijayang) [inline modules/ROOT/pages/create-third-party.adoc]: Please add this note here:

"This is an optional field. If the 3rd party connection uses AS2 protocol, you can configure the AS2 identifier."

File: modules/ROOT/pages/delete-third-party.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
+= Delete a 3rd Party Connection
+
+If you are no longer using a 3rd party connection, you can delete it from Partner Manager.
+
+When you delete a 3rd party connection, it is removed from the *Partners* page. Also, all of the connection's data is deleted, except for information about connection-related activities that occurred before the deletion.
+
```
Comment (vijayang) [inline modules/ROOT/pages/delete-third-party.adoc:6]: Please add a note here:

"You will not be able to delete 3rd party connections if you have configured or deployed message flows for your partners that reference the 3rd party identifier or endpoints."

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 4]
```diff
@@ -1,82 +1,77 @@
 = AS2 Send Endpoint Settings
 
-Use these to set up AS2 endpoint settings to send outbound EDI files to your partners.
+Your organization's AS2 send endpoint settings configure the target endpoint in an outbound message flow to which translated B2B/EDI messages are send to partners either directly or via 3rd party connections.
 
-Create the AS2 identifiers for your organization and your partners in the respective host and partner profile pages.
+A host or partner can own AS2 send endpoints. Use a host-owned AS2 send endpoint for an outbound message flow that uses a 3rd party connection.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Please change to:
"A partner or 3rd party can own AS2 send endpoints. Use a 3rd party-owned AS2 send endpoint for an outbound message flows for partners that use a 3rd party connection.

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 5]
```diff
@@ -1,82 +1,77 @@
...
+
+Create AS2 endpoints by configuring these settings on the Host or Partner pages:
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Please change to:
Create AS2 endpoints by configuring these settings on the Partner or 3rd party connection pages:

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 6]
```diff
@@ -1,82 +1,77 @@
...
 
 [%header%autowidth.spread]
 |===
 |Setting |Description |Required
 
 |Owner
-|The partner or host that owns the endpoint.
+|Partner or host that owns the endpoint.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Partner or 3rd party that owns the endpoint.

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 7]
```diff
@@ -1,82 +1,77 @@
...
 | Yes
 
 |Description
-|The user-defined field for describing the purpose and use of the endpoint.
+|User-supplied description of the endpoint's purpose and use.
 | No
 
 |Partner AS2 URL
-|Your trading partner’s AS2 URL to which outbound EDI transmissions are sent.
+|Your partner’s AS2 URL to which outbound EDI transmissions are sent.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Your partner or 3rd party connection's AS2 URL to which outbound EDI transmissions are sent.

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 8]
```diff
@@ -1,82 +1,77 @@
...
 |Yes
 
 |Retry interval
-|The frequency of reconnection attempts when unable to connect to the remote AS2 service. The value in this field is set to `30 Seconds` by default.
+|Frequency of reconnection attempts when unable to connect to the remote AS2 service. The default is `30 Seconds`.
 |Yes
 
 |Maximum retry attempts
-|The maximum number of reconnection attempts to make.
+|Maximum number of reconnection attempts to make.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Add the below note.

Note: If you do not want Partner Manager to retry after an initial connection failure, set this value to 0.

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 9]
```diff
@@ -1,82 +1,77 @@
...
 |Yes
 
 |<Host> AS2 identifier
 |Your organization’s AS2 identifier.
 |Yes
 
 |Partner AS2 identifier
-|Your trading partner’s AS2 identifier.
+|Your partner’s AS2 identifier.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Your partner or 3rd party connection’s AS2 identifier.

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 10]
```diff
@@ -1,82 +1,77 @@
...
 |Yes
 
 |Message subject
-|The user-defined field for MIME Subject header.
+|User-defined field for the MIME Subject header.
 |Yes
 
 |Filename pattern
-|xref:file-name-pattern.adoc[File name pattern] as required by your partner or backend application.
+|xref:file-name-pattern.adoc[File name pattern] required by your partner or backend application.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Please change to (as AS2 send endpoints do not apply to sending to backend)
"required by your partner"

File: modules/ROOT/pages/endpoint-ftp-receive.adoc [hunk 11]
```diff
@@ -1,68 +1,72 @@
...
 | Yes
 
 | Port
-| The port number of the FTP server.
+| Port number of the FTP server.
 | Yes
 
 | Username
-| The username for the FTP server.
+| Username of the FTP server.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-ftp-receive.adoc]: Please change to:
"Username to connect to the FTP server"

File: modules/ROOT/pages/endpoint-ftp-send.adoc [hunk 12]
```diff
@@ -1,13 +1,18 @@
 = FTP Send Endpoint Settings
 
-You can use these settings to set up FTP endpoints in one of the following contexts:
+Your organization's FTP send endpoint settings configure where to send messages in the following contexts:
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-ftp-send.adoc]: I think we should remove "Your organization's".

File: modules/ROOT/pages/endpoint-https-receive.adoc [hunk 13]
```diff
@@ -1,67 +1,74 @@
 = HTTP and HTTPS Receive Endpoint Settings
 
-Configure HTTP and HTTPS receive endpoints to receive application messages (JSON or XML format) from your backend applications in outbound message flows.
+Your organization's HTTP and HTTPS receive endpoint settings configure where to receive messages in the following contexts:
+
+* Receive from partners
++
+Source endpoint in an inbound message flow that receives B2B/EDI messages from partners either directly or via 3rd party connections
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-https-receive.adoc]: Please remove this. We do not support HTTP and HTTPS as a "Receive from partners" protocol.

File: modules/ROOT/pages/endpoint-https-receive.adoc [hunk 14]
```diff
@@ -1,67 +1,74 @@
...
+* Source at host
++
+Source endpoint in an outbound message flow that receives application JSON and XML messages from the backend systems
+
+A host, partner, or 3rd party connection can own HTTP and HTTPS receive endpoints.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-https-receive.adoc]: Source at host endpoints are always owned by the host.

File: modules/ROOT/pages/endpoint-https-receive.adoc [hunk 15]
```diff
@@ -1,67 +1,74 @@
...
+
+Create HTTP and HTTPS receive endpoints by configuring these settings on the host or Partner page:
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-https-receive.adoc]: Please change to:
"Create HTTP and HTTPS receive endpoints by configuring these settings on the host page"

File: modules/ROOT/pages/endpoint-https-receive.adoc [hunk 16]
```diff
@@ -1,67 +1,74 @@
...
 |===
 |Name |Description |Required
 |Endpoint Name
-|The automatically generated name to identify the endpoint, which you can subsequently modify manually.
+|Automatically-generated name to identify the endpoint. You can modify this value later on.
 |Yes
 
 | Owner
-| The partner or host that owns the endpoint.
+| Partner, 3rd party connection, or host that owns the endpoint.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-https-receive.adoc]: HTTP and HTTPS receive endpoints are always owned by the host.

File: modules/ROOT/pages/endpoint-https-receive.adoc [hunk 17]
```diff
@@ -1,67 +1,74 @@
...
 | Yes
 
 | Protocol
-| The protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window. The value in this field cannot be modified.
+| Protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window. You cannot modify this value.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-https-receive.adoc]: Please change to:
Protocol selected in the *Protocol* field

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 18]
```diff
@@ -1,49 +1,53 @@
 = HTTP and HTTPS Send Endpoint Settings
 
-The HTTP and the secure HTTPS protocol send settings are used with endpoints for sending transmissions to your partner and endpoints that are targets at your organization.
+Your organization's HTTP and HTTPS send endpoint settings configure where to send messages in the following contexts:
 
-You can use the HTTP and HTTPS endpoint to:
+* Send to partners
++
+Target endpoint in an outbound message flow to which translated B2B/EDI messages are sent to the partners
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-https-send.adoc]: Target endpoint in an outbound message flow to which translated B2B/EDI messages are sent to the partners directly or via a 3rd party connection

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 19]
```diff
@@ -1,49 +1,53 @@
...
++
+* Target at host
++
+Target endpoint in an inbound message flow to which translated application JSON and XML messages are sent to the backend systems
 
-* Send to partners +
-Send outbound EDI files to your trading partners in outbound message flows.
-* Target at host +
-Send application messages (JSON or XML) to your backend applications in inbound message flows.
+A host or partner can own HTTP and HTTPS send endpoints.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-https-send.adoc]: A host, 3rd party or partner can own HTTP and HTTPS send endpoints.

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 20]
```diff
@@ -1,49 +1,53 @@
...
+
+Create HTTP and HTTPS receive endpoints by configuring these settings on the host or Partner page:
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-https-send.adoc]: Create HTTP and HTTPS send endpoints by configuring these settings on the host, Partner, or 3rd party connection page:

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 21]
```diff
@@ -1,49 +1,53 @@
...
 
 [%header%autowidth.spread]
 |===
 |Name |Description |Required
 | Owner
-| The trading partner or host that owns the endpoint.
+| Partner or host who owns the endpoint
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-https-send.adoc]: Partner, 3rd party or host who owns the endpoint

File: modules/ROOT/pages/endpoint-sftp-send.adoc [hunk 22]
```diff
@@ -1,13 +1,18 @@
...
-You can use these settings to set up SFTP endpoints in one of the following contexts:
+Your organization's SFTP send endpoint settings configure where to send messages in the following contexts:
 
-* Send to partners +
-Send outbound EDI files to your trading partners in outbound message flows.
-* Target at host +
-Send application messages (JSON or XML) to your backend applications in inbound message flows.
+* Send to partners
++
+Target endpoint in an outbound message flow to which translated B2B/EDI messages are sent to the partners
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-sftp-send.adoc]: Target endpoint in an outbound message flow to which translated B2B/EDI messages are sent to the partners directly or via a 3rd party connection

File: modules/ROOT/pages/create-inbound-message-flow.adoc [hunk 23]
```diff
@@ -1,40 +1,50 @@
...
-... Click *New* next to the search box.
-... From the *Owner* drop-down, select the endpoint's owner.
-... From the *Protocol* drop-down, select the protocol and configure it according to the protocol you select:
-* xref:endpoint-as2-receive.adoc[AS2]
-* xref:endpoint-ftp-receive.adoc[FTP]
-* xref:endpoint-sftp-receive-target.adoc[SFTP]
 +
-... Click *Save*. +
-The endpoint is added to the message flow.
+* Click *Add* to add a unique identifier for the endpoint.
```
Comment (vijayang) [inline modules/ROOT/pages/create-inbound-message-flow.adoc]: Please remove this line, as there is no "Add" step in the actual dialogue.

File: modules/ROOT/pages/create-inbound-message-flow.adoc [hunk 24]
```diff
@@ -1,40 +1,50 @@
...
```
Comment (vijayang) [inline modules/ROOT/pages/create-inbound-message-flow.adoc:52]: Please add the new section below (allowing selection of 3rd party AS2 Sender).
https://docs.google.com/document/d/1K__eyTI2TYpE9HGGYzCMfXpx1mNKf99H9s_ss90o7gA/edit#heading=h.nx6vp3px5yrc

**AS2 Sender**

- If the protocol of the Receive endpoint is AS2, select the AS2 Sender from the dropdown menu:
- If your organization will be receiving the AS2 messages directly from the partner for whom the message flow is created, you can select the partner’s name (Default).
- If your organization will be receiving the AS2 messages for the partner via a 3rd party connection such as a VAN or 3PL, you can select the name of the 3rd party connection. The AS2 identifier and certificate of the selected 3rd party connection will be used in AS2 message validation.
- If you selected AS2 as the Receive to partner endpoint protocol, you will need to import the AS2 certificate of the partner or the 3rd party connection that will be sending the AS2 message, if a certificate already does not exist in the partner or 3rd party connection’s profile.

**Review by vijayang — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/endpoint-ftp-send.adoc (hunk 12 above)
Comment (dhirsch1) [reply modules/ROOT/pages/endpoint-ftp-send.adoc]: I have to remove it from all of the endpoint topics then for consistency, which I don't have an issue with.

File: modules/ROOT/pages/endpoint-https-receive.adoc (hunk 14 above)
Comment (dhirsch1) [reply modules/ROOT/pages/endpoint-https-receive.adoc]: Writing "Only a host can own HTTP and HTTPS receive endpoints."

File: modules/ROOT/pages/endpoint-https-receive.adoc (hunk 17 above)
Comment (dhirsch1) [reply modules/ROOT/pages/endpoint-https-receive.adoc]: We should say where the Protocol field is that it is pulling from.

File: modules/ROOT/pages/create-inbound-message-flow.adoc (hunk 24 above)
Comment (dhirsch1) [reply modules/ROOT/pages/create-inbound-message-flow.adoc:52]: I guess this is a special case under Specify the Receiving Endpoint section? For now I am just going to put it in, but want to circle back to it next week.

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (dhirsch1) [reply modules/ROOT/nav.adoc]: I removed the Configure Send and Receive Settings for Partners and moved the inbound/outbound settings topics back under Configure and Manage Trading Partners in the TOC.

File: modules/ROOT/pages/create-endpoint.adoc [hunk 25]
```diff
@@ -1,60 +1,55 @@
...
+. In the *New Endpoint* window *Usage* field, select *Receive from partners* or *Send to partners*.
 . In the *Protocol* field, select the protocol to use for the endpoint.
 . <<configure-endpoint,Configure the endpoint>> according to the type of endpoint you are creating and the protocol you selected.
 . Click *Save*.
 
-Ownership of the endpoints (receive or send) is automatically assigned based on the partner from which you are creating the endpoint.
+Ownership of the endpoints (receive or send) is automatically assigned based on the partner or 3rd party connection from which you are creating the endpoint.
 
-== Create and Configure Endpoints from the Host Page
+== Create Endpoints from the Host page
```
Comment (sathya0) [inline modules/ROOT/pages/create-endpoint.adoc]:
```suggestion
== Create Endpoints from the Host Page
```

File: modules/ROOT/pages/create-inbound-message-flow.adoc [hunk 26]
```diff
@@ -1,40 +1,56 @@
...
 === Steps
 
-. From your Anypoint Partner Manager sandbox or production environment, select *Message Flows* in the navigation menu.
+. From the Anypoint Partner Manager navigation menu, select *Message Flows*.
 . Click *New message flow > Receive from partner*.
-. Either select an existing partner from the list and click *Select* or select *New* to xref:configure-partner.adoc#create-partner[create a trading partner]. +
-A new message flow is created. +
+. Either select an existing partner from the list or select *New* to create a new partner, as described in xref:create-partner.adoc#[Create a partner].
++
+A new message flow is created.
```
Comment (sathya0) [inline modules/ROOT/pages/create-inbound-message-flow.adoc:17]: Is this formatted right? The "A new message flow is created."

File: modules/ROOT/pages/create-inbound-message-flow.adoc [hunk 27]
```diff
@@ -1,40 +1,56 @@
...
-... Click *Add* to add a unique identifier for the endpoint.
-.. To create a new receiving endpoint:
-... Click *New* next to the search box.
-... From the *Owner* drop-down, select the endpoint's owner.
-... From the *Protocol* drop-down, select the protocol and configure it according to the protocol you select:
-* xref:endpoint-as2-receive.adoc[AS2]
-* xref:endpoint-ftp-receive.adoc[FTP]
-* xref:endpoint-sftp-receive-target.adoc[SFTP]
+. In the *Select an endpoint: Receive from <partner>* window, do either of the following:
+* Select an existing receiving endpoint and click  *Select*.
```
Comment (sathya0) [inline modules/ROOT/pages/create-inbound-message-flow.adoc]:
```suggestion
* Select an existing receiving endpoint and click *Select*.
```

File: modules/ROOT/pages/create-inbound-message-flow.adoc [hunk 28]
```diff
@@ -1,40 +1,56 @@
...
+* Click *New* to create a new endpoint, as described in xref:<<create-endpoint.adoc, Create and Configure Endpoints>>.
```
Comment (sathya0) [inline modules/ROOT/pages/create-inbound-message-flow.adoc:41]:
```suggestion
* Click *New* to create a new endpoint, as described in xref:create-endpoint.adoc[Create and Configure Endpoints].
```

File: modules/ROOT/pages/create-outbound-message-flow.adoc [hunk 29]
```diff
@@ -49,7 +49,7 @@ The message type is added to the message flow.
 
 . Expand the *Source at <host>* section.
 . From the *<host> reference ID* drop-down list, select your organization’s internal reference ID (such as business unit name) as it would appear in the data element of the source JSON or XML payload from your backend systems. +
-The reference ID is added to the message flow.
+The reference ID is added to the message flow
```
Comment (sathya0) [inline modules/ROOT/pages/create-outbound-message-flow.adoc]:
```suggestion
The reference ID is added to the message flow.
```

File: modules/ROOT/pages/create-outbound-message-flow.adoc [hunk 30]
```diff
@@ -94,6 +94,27 @@ By default, Partner Manager uses the X12 send or EDIFACT send settings that are
 To apply custom validation settings and delimiters for the message flow you are setting up, select *Custom X12 settings* or *Custom EDIFACT settings* and provide custom configuration values.
 . In the *<protocol> Identifiers* section, select the sender and receiver identifiers that are configured under the host and trading partner profiles to use them on the X12 ISA and GS segments or the EDIFACT UNB segments.
 
+== Configure the Target endpoint
```
Comment (sathya0) [inline modules/ROOT/pages/create-outbound-message-flow.adoc]:
```suggestion
== Configure the Target Endpoint
```

File: modules/ROOT/pages/create-partner.adoc [hunk 31]
```diff
@@ -0,0 +1,73 @@
...
+|Information about the partner, such as the partner's address, website, and contact details.
+| Click *Edit* in the *About* section.
+|
+| Additional identifiers to associate with the partner.
+| Click *New* in the *Identifiers* section.
+| xref:partner-manager-identifiers.adoc[Sender and Receiver Identifiers]
+| The partner's digital certificates, such as X.509 certificates.
+| Click *New* in the *Certificates* section.
+|
+| The partner-specific endpoints to use for sending and receiving messages to and from your partner, including acknowledgement endpoints. If you plan to use host endpoints only, you don't need to create partner-specific endpoints.
```
Comment (sathya0) [inline modules/ROOT/pages/create-partner.adoc:53]:
```suggestion
| The partner-specific endpoints to use for sending and receiving messages to and from your partner, including acknowledgment endpoints. If you plan to use host endpoints only, you don't need to create partner-specific endpoints.
```

File: modules/ROOT/pages/create-partner.adoc [hunk 32]
```diff
@@ -0,0 +1,73 @@
...
+| Click *New* in the *Endpoints* section.
+| xref:create-endpoint.adoc[Create and Configure Endpoints]
+| Acknowledgements, validations, and control numbers for each message format that the partner uses. For X12, you can also configure character sets and separators (X12 send format only).
+| Click the links in the *Validation and acknowledgement* settings.
```
Comment (sathya0) [inline modules/ROOT/pages/create-partner.adoc:57]:
```suggestion
| Click the links in the *Validation and acknowledgment* settings.
```

File: modules/ROOT/pages/edifact-receive-read-settings.adoc [hunk 33]
```diff
@@ -19,62 +19,82 @@ It can take up to five minutes or so for your configuration changes to sync with
 [[ack-settings]]
 == Acknowledgment Settings
 
-To configure acknowledgements:
+Optionally configure acknowledgment settings for each Inbound EDIFACT message flow. To configure acknowledgement settings:
```
Comment (sathya0) [inline modules/ROOT/pages/edifact-receive-read-settings.adoc:22]:
```suggestion
Optionally configure acknowledgment settings for each Inbound EDIFACT message flow. To configure acknowledgment settings:
```

File: modules/ROOT/pages/edifact-receive-read-settings.adoc [hunk 34]
```diff
@@ -19,62 +19,82 @@ It can take up to five minutes or so for your configuration changes to sync with
...
 
-. Enable the *Send CONTRL functional acknowledgement when messages fail to route to a message flow* setting.
-. Select an endpoint where the functional acknowledgement message will be sent to the trading partner.
+* Enable the *Send CONTRL functional acknowledgement when messages fail to route to a message flow* setting.
```
Comment (sathya0) [inline modules/ROOT/pages/edifact-receive-read-settings.adoc:24]:
```suggestion
* Enable the *Send CONTRL functional acknowledgment when messages fail to route to a message flow* setting.
```

File: modules/ROOT/pages/edifact-receive-read-settings.adoc [hunk 35]
```diff
@@ -19,62 +19,82 @@ It can take up to five minutes or so for your configuration changes to sync with
...
++
+When there is a routing failure, this acknowledgement indicates that the message is rejected.
```
Comment (sathya0) [inline modules/ROOT/pages/edifact-receive-read-settings.adoc:26]:
```suggestion
When there is a routing failure, this acknowledgment indicates that the message is rejected.
```

File: modules/ROOT/pages/edifact-send-settings.adoc [hunk 36]
```diff
@@ -43,6 +43,20 @@ Use the following settings to configure the general validation rules:
 |Disabled
 |===
 
+[[global-control-numbers]]
+== Global Control Numbers
+
+Partner Manager applies a global control number at the partner level on the EDIFACT interchanges generated from outbound message flows. These control numbers are sequenced across all message types sent to a trading parter, including functional acknowledgments.
```
Comment (sathya0) [inline modules/ROOT/pages/edifact-send-settings.adoc]:
```suggestion
Partner Manager applies a global control number at the partner level on the EDIFACT interchanges generated from outbound message flows. These control numbers are sequenced across all message types sent to a trading partner, including functional acknowledgments.
```

File: modules/ROOT/pages/create-inbound-message-flow.adoc (hunk 28 above)
Comment (dhirsch1) [reply modules/ROOT/pages/create-inbound-message-flow.adoc:41]: Great catch. I don't think the link checker caught this.

File: modules/ROOT/pages/create-partner.adoc (hunk 31 above)
Comment (dhirsch1) [reply modules/ROOT/pages/create-partner.adoc:53]: You are correct that this is a British spelling, but that is how the GUI has it, so I am leaving it at least for now. (I will ask Kim about this next week.)

File: modules/ROOT/pages/create-partner.adoc (hunk 32 above)
Comment (dhirsch1) [reply modules/ROOT/pages/create-partner.adoc:57]: This is the GUI label.

File: modules/ROOT/pages/create-inbound-message-flow.adoc (hunk 26 above)
Comment (dhirsch1) [reply modules/ROOT/pages/create-inbound-message-flow.adoc:17]: It works, even though I think it is better to have the + on its own line. I am changing these as I rewrite a topic.

**Review by sathya0 — APPROVED** [review]

**Review by vijayang — APPROVED** [review]

**Pull request merged**